src/ngraph/runtime/cpu/CMakeLists.txt

```diff
@@ -157,5 +149,12 @@ if (NGRAPH_CPU_ENABLE)
     target_include_directories(cpu_backend SYSTEM PUBLIC libmkldnn)
     set_target_properties(cpu_backend PROPERTIES LIBRARY_OUTPUT_DIRECTORY ${NGRAPH_BUILD_DIR})
 
+    if(NGRAPH_DISTRIBUTED_ENABLE)
+        add_definitions(-DNGRAPH_DISTRIBUTED)
+        add_dependencies(cpu_backend libmlsl)
```

Comment:
not needed

Reply:
Removed

Reply:
@AlekseyMarchuk FYI, there's no need to reply to the comment about the code you modified based on the review. Github will hide this after your commit so reviewers know you have made the change. :)